### DISCUSSION TITLE
MONAI not recognizing ITK

### DISCUSSION THREAD

**Discussion by santiago-pires** [Project-MONAI/MONAI#6462] [Q&A]
I have monai:
MONAI version: 0.8.0
Numpy version: 1.24.2
Pytorch version: 1.12.1+cu116
MONAI flags: HAS_EXT = False, USE_COMPILED = False
MONAI rev id: 714d00dffe6653e21260160666c4c201ab66511b

Optional dependencies:
Pytorch Ignite version: 0.4.8
Nibabel version: NOT INSTALLED or UNKNOWN VERSION.
scikit-image version: NOT INSTALLED or UNKNOWN VERSION.
Pillow version: 9.3.0
Tensorboard version: 2.11.0
gdown version: 4.2.1
TorchVision version: 0.13.1+cu116
tqdm version: 4.64.1
lmdb version: 1.3.0
psutil version: 5.9.0
pandas version: 1.5.2
einops version: 0.3.0
transformers version: 4.16.2
mlflow version: 1.23.1

import itk
print(itk.__version__) -> 5.2.1

**from monai.data import ITKReader

itk_reader = ITKReader()**

Error:
OptionalImportError                       Traceback (most recent call last)
/tmp/ipykernel_130853/517721973.py in <cell line: 3>()
      1 from monai.data import ITKReader
      2 
----> 3 itk_reader = ITKReader()

~/jupyter_dir/jupyter_env/lib/python3.8/site-packages/monai/utils/module.py in _wrapper(*args, **kwargs)
    379                 err_msg = f"required package `{pkg_name}` is not installed or the version doesn't match requirement."
    380                 if raise_error:
--> 381                     raise OptionalImportError(err_msg)
    382                 else:
    383                     warnings.warn(err_msg)

OptionalImportError: required package `itk` is not installed or the version doesn't match requirement.

I tried re-installing packages, monai, etc. Nothing works, still getting the same error. I would appreciate any insights into this : )

**Comment by wyli** [comment]
`MONAI version: 0.8.0` `MONAI rev id: 714d00d` suggest there are different versions of monai installed... please make sure you have at least python 3.8 and uninstall all the previous versions of monai then install https://docs.monai.io/en/latest/installation.html
